src/components/form-slice/FormPartSecret.tsx

```diff
@@ -0,0 +1,172 @@
+        name="prefix"
+        label={t('form.secrets.vault.prefix')}
+      />
+      <FormItemTextInput
+        control={control}
+        name="token"
+        label={t('form.secrets.vault.token')}
+      />
+      <FormItemTextInput
+        control={control}
+        name="namespace"
+        label={t('form.secrets.vault.namespace')}
+      />
+    </>
+  );
+};
+
+const AWSSecretForm = () => {
+  const { t } = useTranslation();
+  const { control } = useFormContext<APISIXType['Secret']>();
+
+  return (
+    <>
+      <FormItemTextInput
+        control={control}
+        name="access_key_id"
+        label={t('form.secrets.aws.access_key_id')}
+      />
+      <FormItemTextInput
+        control={control}
+        name="secret_access_key"
+        label={t('form.secrets.aws.secret_access_key')}
+      />
+      <FormItemTextInput
+        control={control}
+        name="session_token"
+        label={t('form.secrets.aws.session_token')}
+      />
+
+      <FormItemTextInput
+        control={control}
+        name="region"
+        label={t('form.secrets.aws.region')}
+      />
+      <FormItemTextInput
+        control={control}
+        name="endpoint_url"
+        label={t('form.secrets.aws.endpoint_url')}
+      />
+    </>
+  );
+};
+
+const GCPSecretForm = () => {
+  const { t } = useTranslation();
+  const { control } = useFormContext<APISIXType['Secret']>();
+
+  return (
+    <>
+      <InputWrapper label={t('form.secrets.gcp.ssl_verify')}>
+        <FormItemSwitch control={control} name="ssl_verify" />
+      </InputWrapper>
+      <FormSection legend={t('form.secrets.gcp.auth')}>
+        <FormItemTextInput
+          control={control}
+          name="auth_file"
+          label={t('form.secrets.gcp.auth_file')}
+        />
+        <Divider my="xs" label={t('or')} />
+        <FormSection legend={t('form.secrets.gcp.auth_config')}>
+          <FormItemTextInput
+            control={control}
+            name="auth_config.client_email"
+            label={t('form.secrets.gcp.client_email')}
+          />
+          <FormItemTextInput
+            control={control}
+            name="auth_config.private_key"
+            label={t('form.secrets.gcp.private_key')}
+          />
+          <FormItemTextInput
+            control={control}
+            name="auth_config.project_id"
+            label={t('form.secrets.gcp.project_id')}
+          />
+          <FormItemTextInput
+            control={control}
+            name="auth_config.token_uri"
+            label={t('form.secrets.gcp.token_uri')}
+          />
+          <FormItemTagsInput
+            control={control}
+            name="auth_config.scope"
+            label={t('form.secrets.gcp.scope')}
+          />
+          <FormItemTextInput
+            control={control}
+            name="auth_config.entries_uri"
+            label={t('form.secrets.gcp.entries_uri')}
+          />
+        </FormSection>
+      </FormSection>
+    </>
+  );
+};
```

Comment:
Is it necessary to add a specialized form? I think using an editor with syntax highlighting and jsonschema would be sufficient.
If it doesn't exist, we'll have to add the secret provider's jsonschema export.

Reply:
I think it's necessary at the moment. 

If it's needed as you said, then secrets should provide options for export and schema export, just like the way plugins do it. But these things don't exist yet, we should discuss and add these features in APISIX.

Reply:
This needs to be done before the release so no forms that shouldn't be there are added to artifacts, approved and merged with it.